### PR TITLE
Fix#72 Add Voltage Decibel Functions

### PR DESCRIPTION
**Pull request by khan-asfi-reza**
Added 3 Functions as per the issue #72

### CODE REVIEW

File: electricpy/conversions.py [hunk 1]
```diff
@@ -569,4 +569,73 @@ def watts_to_dbmw(watts):
...
+                 voltage
+
+    ref_voltage: float
+                 Reference voltage
+    Return
+    ------
+    decibel:    float
+                voltage in the decibel scale
+    """
+
```
Comment (engineerjoe440) [inline electricpy/conversions.py:593]: This blank-line is upsetting the `pydocstyle` automatic CI (basically enforcing "best-practices" in the documentation of the library), if we remove this line (593) then everything should be good-to-go! :+1:

**Review by engineerjoe440 — CHANGES_REQUESTED** [review]
Just some documentation "complaints" :stuck_out_tongue: Should be pretty easy to fix. Thank you!

File: electricpy/conversions.py (hunk 1 above)
Comment (khan-asfi-reza) [reply electricpy/conversions.py]: Fixed the blank line

**Review by engineerjoe440 — APPROVED** [review]
Sweet! Looks good to me!

**Pull request merged**